src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class RightTriangle extends AbstractFigure {
+    private double base;
+    private double heigth;
+
+    public RightTriangle(String color,double base, double heigth) {
+        super(color);
+        this.base = base;
+        this.heigth = heigth;
+    }
+
+    public double getArea() {
+        return 0.5 * base * heigth;
+    }
+
+    @Override
+    public String describe() {
+        return "RightTriange " + getColor() + " base: " + base + " heigth: "
+                + heigth + " area: " + getArea();
```

Comment:
The variable `heigth` is misspelled. It should be `height` to maintain code clarity and prevent potential bugs.